laboratory/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
```

Comment:
We don't have main and master, instead it's V2. Assuming same tests run during pull request, do we need them to re-run on push?